.github/workflows/ui-tests.yml

```diff
@@ -105,3 +105,5 @@ jobs:
 
       - name: UI Tests
         run: cd ~/frappe-bench/ && bench --site test_site run-ui-tests frappe --headless --parallel --ci-build-id $GITHUB_RUN_ID
+        env:
+          CYPRESS_RECORD_KEY: 4a48f41c-11b3-425b-aa88-c58048fa69eb
```

Comment:
github secrets?

Reply:
Yes! ![image](https://user-images.githubusercontent.com/16315650/124065411-674ade80-da54-11eb-98af-47dd46c5a541.png)

Reply:
This is the recommended way: https://docs.cypress.io/guides/dashboard/github-integration#Install-the-Cypress-GitHub-app

Reply:
My concern was not billing for now(it might be  valid concern). I just meant to hide the key using a GitHub secret(this still allows people out of the organisation to run tests)

Reply:
Yes, it will unnecessarily pollute the dashboard.
If possible, we need to generate a new one.

Reply:
github secrets are not passed on to forks AFAIK. Is there a better way?

I did add it as GitHub secret in ERPNext repo (as it doesn't matter much right now if it failes, there aren't any tests)

It doesn't work on PRs from fork, example: https://github.com/frappe/erpnext/pull/26265/checks?check_run_id=2955142472

Reply:
https://github.community/t/make-secrets-available-to-builds-of-forks/16166

Reply:
But hey, won't cypress running on people's forks get recorded as well?

Reply:
Yeah, as long as it matches the criteria for branches 👀 

That can be refined I guess.